contracts/Removal.sol

```diff
@@ -165,28 +165,62 @@ contract Removal is
   }
 
   /**
-   * @notice Marks an amount of a removal as released given a removal ID and a quantity.
-   * @param owner The owner of the removal to release.
-   * @param removalId The ID of the removal to mark as released.
-   * @param amount The amount of the removal to release.
-   *
-   * @dev
+   * @notice Releases an amount of a removal.
+   * @dev Releases `amount` of removal by `removalId` by burning them.
    *
    * ##### Requirements:
    *
+   * - Releasing burns first from unlisted balances, second from listed balances and third from certificates.
+   * - If the removal is unlisted (e.g., owned by any account other than the market or certificate), the removal is
```

Comment:
```suggestion
   * - If the removal is unlisted (e.g., owned by the supplier account encoded in the removal id and not the market or certificate contracts), the removal is
```

Reply:
I think I prefer the original as it's still correct and future proof